bitbots_ros_control/include/bitbots_ros_control/dynamixel_servo_hardware_interface.h

```diff
@@ -58,35 +58,35 @@ struct Joint
 };
 
 enum ControlMode {
-  PositionControl,
-  VelocityControl,
-  EffortControl,
-  CurrentBasedPositionControl
+  POSITION_CONTROL,
+  VELOCITY_CONTROL,
+  EFFORT_CONTROL,
+  CURRENT_BASED_POSITION_CONTROL
 };
 
 class DynamixelServoHardwareInterface : public hardware_interface::RobotHW
 {
 public:
   DynamixelServoHardwareInterface();
-  DynamixelServoHardwareInterface(std::shared_ptr<DynamixelDriver>& driver);
-  void reconf_callback(bitbots_ros_control::dynamixel_servo_hardware_interface_paramsConfig &config, uint32_t level);
+  explicit DynamixelServoHardwareInterface(std::shared_ptr<DynamixelDriver>& driver);
+  void reconfCallback(bitbots_ros_control::dynamixel_servo_hardware_interface_paramsConfig &config, uint32_t level);
 
   bool init(ros::NodeHandle& nh);
   bool read();
   void write();
   void setParent(hardware_interface::RobotHW* parent);
 
 private:
-  ros::NodeHandle _nh;
+  ros::NodeHandle nh_;
 
-  void syncWritePWM();
+  void syncWritePwm();
```

Comment:
Since PWM is an abbreviation, i would keep it as syncWritePWM

Reply:
okay